README.markdown

```diff
@@ -30,7 +30,8 @@ or Wget:
 
     wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub>
+<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub> 
+Eg. `source ~/.profile`
```

Comment:
On ubuntu, you need to use "manual install". If you're using the install script, you shouldn't need to know how it changes your profile, because it's handling it for you.